dev-tools/omdb/src/bin/omdb/db.rs

```diff
@@ -2499,13 +2499,10 @@ async fn cmd_db_inventory_collections_show(
     limit: NonZeroU32,
     long_string_formatter: LongStringFormatter,
 ) -> Result<(), anyhow::Error> {
-    let (collection, incomplete) = datastore
-        .inventory_collection_read_best_effort(opctx, id, limit)
+    let collection = datastore
+        .inventory_collection_read_batched(opctx, id, limit)
```

Comment:
Related to my other comment: I don't think it's important that `omdb` pass through its `limit` argument here.  That argument exists for the many cases that _don't_ do pagination because there the `omdb` user has to make a tradeoff between completeness and (in theory) risk of making a super expensive query.  In cases that are doing client-side pagination, we don't have that tradeoff (hooray!) so it's fine to ignore that parameter.